src/mlpack/bindings/python/tests/test_python_binding.py

```diff
@@ -192,7 +192,7 @@ def testPandasSeriesMatrix(self):
     Test that we can pass pandas.Series as input parameter.
     """
     x = pd.Series(np.random.rand(100))
-    z = copy.copy(x)
```

Comment:
`copy.copy` does a shallow copy. `copy.deepcopy` is needed for the above. I guess may  be that was the reason.

Reply:
All set. Good to go.

Reply:
Thanks, I must have misread the documentation.  Fixed now. :+1: